jenkins_jobs/build_host_os/script.sh

```diff
@@ -29,8 +24,6 @@ fi
 # Fetch pull requests in case this job was triggered by one
 git fetch origin +refs/pull/*:refs/remotes/origin/pr/*
```

Comment:
It is removed in the next commit: "Do not clone versions repository"
I thought separating those commits would make it easier to see what exactly was removed and what just changed place.